fastlane/Fastfile

```diff
@@ -539,6 +533,16 @@ platform :android do
     )
   end
 
+  lane :create_backmerge_pr do |version: release_version_current|
+    create_release_backmerge_pull_request(
+      repository: GH_REPOSITORY,
+      source_branch: "release/#{version}",
+      default_branch: DEFAULT_BRANCH,
```

Comment:
Ah yes, this is required here as the default branch for PCAndroid is `main` 👍

Reply:
Yep, didn't have it at first but noticed it was necessary during testing 😅 

Note that I found out the other day that the `Git` ruby gem has a [`repository_default_branch` method](https://github.com/ruby-git/ruby-git/blob/master/lib/git/base.rb#L28) that maybe we could start using in our actions of `release-toolkit` to avoid the need for those kind of parameters in the future 🤔